.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "apollographql.vscode-apollo",
```

Comment:
I _think_ we should use the Apollo extension instead of [the official GraphQL extension](https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql).